src/main/scala/it/pagopa/interop/agreementprocess/api/impl/AgreementApiServiceImpl.scala

```diff
@@ -80,10 +106,12 @@ final case class AgreementApiServiceImpl(
 
     val result: Future[Agreement] = for {
       requesterOrgId <- getOrganizationIdFutureUUID(contexts)
-      eService       <- catalogManagementService.getEServiceById(payload.eserviceId)
+      eService       <- catalogManagementService
+        .getEServiceById(payload.eserviceId)
```

Comment:
to reduce noice, can you make this and all other similar changes in one line?
```suggestion
      eService       <- catalogManagementService.getEServiceById(payload.eserviceId)
```